pypeit/specobj.py

```diff
@@ -350,39 +350,67 @@ def copy(self):
         # Return
         return copy.deepcopy(self)
 
-    def flexure_interp(self, sky_wave, fdict):
+    @staticmethod
```

Comment:
Given that this is now being used to interpolate wavelength images, I think this actually should just be a utility method somewhere in core/wave.py since in that context it makes little sense to import Specobs and use a static method on specobjs.